avm/ptn/app-service-lza/hosting-environment/tests/e2e/waf-aligned/main.test.bicep

```diff
@@ -0,0 +1,53 @@
+metadata name = 'WAF-aligned'
+metadata description = 'This instance deploys the module with WAF aligned settings.'
+
+targetScope = 'subscription'
+
+// ========== //
+// Parameters //
+// ========== //
+
+@description('Optional. The location to deploy resources to.')
+param resourceLocation string = deployment().location
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'appwaf'
+
+@description('Optional. Test name prefix.')
+param namePrefix string = '#_namePrefix_#'
+
+@description('Optional. The password to leverage for the login.')
+@secure()
+param password string = newGuid()
+
+// ============== //
+// Test Execution //
+// ============== //
+
+module testDeployment '../../../main.bicep' = {
+  name: '${uniqueString(deployment().name, resourceLocation)}-test-${serviceShort}'
+  params: {
+    workloadName: serviceShort
+    tags: {
+      environment: 'test'
+    }
+    location: resourceLocation
```

Comment:
```suggestion
```

Reply:
Should only contain values needed to be WAF aligned (i.e., tags, diagnostics, security, reliability, etc.)

Reply:
Why was this marked as resolved? The `location` still seems to be configured.